<launch>
	<arg name="calibFile" />
	<!--<arg name="calibFile" value="$(env HOME)/data/lab/calib.xml" />-->
	<!--<arg name="calibFile" value="$(env HOME)/data/flight4/calib2d/calib.xml" />-->

	<arg name="worldFrame" value="/ptam_camera" />
	<arg name="maxFeatures" value="800" />

	<!-- <arg name="extraArgs" value="-g" /> -->
	<arg name="extraArgs" value="" />

	<!--<arg name="launchPrefix" value="gdb -ex run \-\-args" />-->
	<!--<arg name="launchPrefix" value="gdb \-\-args" />-->
	<!--<arg name="launchPrefix" value="valgrind" />-->
	<arg name="launchPrefix" value="" />

	<!-- <node pkg="sparsestereo" type="sparsestereo"
		name="sparsestereo"
		args="-r $(arg calibFile) -M $(arg maxFeatures) $(arg extraArgs)" output="screen"
		launch-prefix="$(arg launchPrefix)"
	/> -->

	<!-- <node pkg="nodelet" type="nodelet" name="manager"  args="manager" output="screen"/> -->
	<node pkg="nodelet" type="nodelet"
		name="sparsestereo"
		args="load sparsestereo/SparseStereoNodelet manager -r $(arg calibFile) -M $(arg maxFeatures) $(arg extraArgs) -W$(arg worldFrame)" output="screen" 
		launch-prefix="$(arg launchPrefix)"
	/>
</launch>
